saleor/core/utils/metadata_manager.py

```diff
@@ -73,3 +74,12 @@ def create_from_graphql_input(
     return MetadataItemCollection(
         [MetadataItem(item.key, item.value) for item in items]
     )
+
+
+def method_metadata_is_valid(metadata: Any) -> bool:
+    if not isinstance(metadata, dict):
+        return False
+    for key, value in metadata.items():
+        if not isinstance(key, str) or not isinstance(value, str) or not key.strip():
```

Comment:
Are you sure that value is always `str`? What happens if someone sends any numeric value?

Reply:
I just moved the method to different place. Prepared a ticket to address the concerns:
https://linear.app/saleor/issue/EXT-2110__